ambry-tools/src/main/java/com.github.ambry/store/DumpIndexTool.java

```diff
@@ -392,11 +441,14 @@ public IndexProcessingResults processIndex(File replicaDir, Set<StoreKey> filter
           StoreKey key = entry.getKey();
           if (filterSet == null || filterSet.isEmpty() || filterSet.contains(key)) {
             boolean isDelete = entry.getValue().isFlagSet(IndexValue.Flags.Delete_Index);
+            boolean isTtlUpdate = !isDelete && entry.getValue().isFlagSet(IndexValue.Flags.Ttl_Update_Index);
```

Comment:
Could do that but it seems to me that the enum would have very limited reach (only this function?) because eventually this just translates into something that is represented by `BlobState` (`Valid`, `Deleted`, `Expired`).